docs/how-to-guides/how-to-create-maps-for-autoware.md

```diff
@@ -0,0 +1,59 @@
+- A map can be saved as a single PCD file or divided into multiple PCD files.
+- Each point in a map must contain X, Y, and Z coordinates.
+- An intensity or RGB value of each point can be included in the PCD file but is optional.
+- A map must be smaller than 1 GB as it is limited by ROS message size.
+- A map resolution should be at least 0.2 m to yield reliable localization results.
+- Although a map can generally be in either local or global coordinates, it must be in global coordinates (georeferenced) if you want to use GNSS data for localization.
+- [Military Grid Reference System (MGRS)](https://en.wikipedia.org/wiki/Military_Grid_Reference_System) is used as a coordinate system for a georeferenced map. In a georeferenced map, the X and Y coordinates of each point represent the point's location within a 100 km MGRS grid, while the Z coordinate represents the point's elevation.
+
+Traditionally, a Mobile Mapping System (MMS) is used in order to create highly accurate large-scale point cloud maps. However, since a MMS requires high-end sensors for precise positioning, its operational cost can be very expensive and may not be suitable for a relatively small driving environment. Alternatively, a Simultaneous Localization And Mapping (SLAM) algorithm can be used to create a point cloud map from recorded LiDAR scans. Commonly used open-source SLAM implementations include [hdl-graph-slam](https://github.com/koide3/hdl_graph_slam) (LiDAR, IMU\*, GNSS\*), [LeGO-LOAM](https://github.com/facontidavide/LeGO-LOAM-BOR) (LiDAR, IMU\*), [LeGO-LOAM-BOR](https://github.com/RobustFieldAutonomyLab/LeGO-LOAM) (LiDAR), [LIO-SAM](https://github.com/TixiaoShan/LIO-SAM) (LiDAR, IMU, GNSS), and [lidarslam-ros2](https://github.com/rsasaki0109/lidarslam_ros2) (LiDAR, IMU\*). The required sensor data for each algorithm is specified in the parentheses, where an asterisk (\*) indicates that such sensor data is optional. Most of these algorithms already have a built-in loop-closure and pose graph optimization. However, if the built-in, automatic loop-closure fails or does not work correctly, you can use [Interactive SLAM](https://github.com/SMRT-AIST/interactive_slam) to adjust and optimize a pose graph manually. For supported LiDAR models, please check the Github repository of each algorithm.
+
+It is important to note that apart from [lidarslam-ros2](https://github.com/rsasaki0109/lidarslam_ros2), these open-source SLAM implementations are based on ROS 1. Therefore, it could be problematic if you want to install them on the same machine that runs Autoware, which is based on ROS 2. To avoid this problem, you can use Docker or install them on a different machine. Another problem is the ROSBAG version, as many of these SLAM implementations require ROSBAG 1 instead of ROSBAG 2 used by Autoware. For the ROSBAG version problem, you may use this [stand-alone converter](https://gitlab.com/MapIV/rosbags) to convert a ROSBAG 2 file to a ROSBAG 1 file and vice versa.
```

Comment:
I think that’s because there are more ROS1-based SLAM implementations that are known to work well and do not strictly require other sensors except LiDAR (at least for now). Also, the ones that use ROS2 don’t seem to be as mature as those that are based on ROS1.

However, I understand that many users may prefer an easy-to-use implementation just to get started quickly. I think maybe if I switch their position and talk about ROS2-based ones first that would help. Essentially, what @LalithVipulananthan  suggested.